### DISCUSSION TITLE
Problem regarding pull request

### DISCUSSION THREAD

**Discussion by ajmi-nishuti** [AstroBridge/BDLensing#167] [Q&A] [answered]
How can I solve this problem ?

![Capture](https://github.com/ajshajib/BDLensing/assets/132552905/dbad531b-0f07-4a10-b747-4cee4e4d81fc)
![Screenshot 2023-07-02 224257](https://github.com/ajshajib/BDLensing/assets/132552905/c90898c9-9075-45b1-913e-e0bbc4cd9843)

**Comment by rafee-adnan** [comment]
You assigned yourself and unassigned yourself, I think that might be creating the problem. My suggestion would be to delete this pull request and open another one.

**Comment by ajshajib** [comment, marked as answer]
That's an error with the automatic reviewer assignment. Your PR is good to be reviewed, no worries. I will look into fixing the problem.

**Reply by ajmi-nishuti** [reply]
Thank you bhai
